staging/src/k8s.io/api/coordination/v1/types.go

```diff
@@ -58,6 +73,17 @@ type LeaseSpec struct {
 	// holders.
 	// +optional
 	LeaseTransitions *int32 `json:"leaseTransitions,omitempty" protobuf:"varint,5,opt,name=leaseTransitions"`
+	// Strategy indicates the strategy for picking the leader for coordinated leader election
+	// A nil Strategy implies no coordination.
```

Comment:
nit: "nil" is a Go-ism.  How about "If this field is not specified, there is no active coordination for this lease".